src/test/relation_engine_client/test_integration.py

```diff
@@ -0,0 +1,137 @@
+import unittest
+import os
+from uuid import uuid4
+
+from src.relation_engine_client import REClient
+from src.relation_engine_client.exceptions import RERequestError, RENotFound
+
+_API_URL = os.environ.get('RE_API_URL', 'http://localhost:5000')
+_VERT_COLL = 'test_vertex'
+_EDGE_COLL = 'test_edge'
+_TOK_ADMIN = 'admin_token'
+_TOK_USER = 'non_admin_token'
+_TOK_INVALID = 'invalid_token'
+
+
+class TestREClientIntegration(unittest.TestCase):
+    """Integration tests for the REClient package."""
+
+    @classmethod
+    def setUpClass(cls):
+        cls.client = REClient(_API_URL, _TOK_ADMIN)
+        pass
+
+    def test_admin_query_ok(self):
+        _id = self._save_test_vert()
+        bind_vars = {'id': _id}
+        query = f"FOR vert IN {_VERT_COLL} FILTER vert._key == @id RETURN vert"
+        result = self.client.admin_query(query, bind_vars)
+        self.assertEqual(result['count'], 1)
+        self.assertEqual(result['results'][0]['_key'], _id)
+
+    def test_admin_empty_query(self):
```

Comment:
Check that message is passed through correctly? This applies to all the tests really